translation/transport_test.go

```diff
@@ -133,12 +133,6 @@ func TestDecodeRequestPartnerIDs(t *testing.T) {
 				ctx = bascule.WithAuthentication(ctxTID, auth)
 			}
 
-			if test.tokenType == "" {
-				ctx = ctxTID
-			} else {
-				ctx = bascule.WithAuthentication(ctxTID, auth)
```

Comment:
Do we not need this?

Reply:
It seems to be repeated code from lines 130 - 134, so I took it out